diffusers/training_example.ipynb

```diff
@@ -0,0 +1,3955 @@
+{
```

Comment:
<p>The language can be a bit more direct. I would just say something like "Diffusion models take in an noisy image and iteratively return a de-noised version". </p><p> </p><p>I think it could be useful to have a little review content in the training colab.</p><ol><li>the base question for the distribution being modeled,</li><li>maybe the same figure used in the readme.md.</li></ol><p> </p><p>Minor nits: can the type class of <code>UNet</code> be fixed to match source? Also, should U-net be hyphenated? Is that how its normally referred to in the literature? Adding a link to the original UNet paper would be very helpful.</p><p> </p><p>Is this doable?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>